ApollosProject/apollos-apps
# fix modal control

## ✏️ Description
this fixes the bug described here: https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/5589831721
This fix was worked on by @calebpanza and myself. We found an override for color on the blurview component. Leveraging it we are able to get this to function as expected. 

## 🔬 To Test
1. clone repo
2. run the app on android sim
3. go to daily habits with sim in light mode
4. open prayer or gratitude daily habit. 
5. modal buttons t at the top should be light now. 

## 📸 Screenshots
fixed
![Screenshot 2022-12-06 at 1 22 02 PM](https://user-images.githubusercontent.com/16566029/206002963-dc7da439-3486-47a9-89fa-652883c201c2.png)


## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
